src/ReachSets/DiscretePost/LazyDiscretePost.jl

```diff
@@ -159,10 +159,7 @@ function post(𝒫::LazyDiscretePost,
             end
 
             # apply assignment
-            A⌜R⋂G⌟ = LinearMap(assignment, R⋂G)
-            if !𝒫.options[:lazy_R⋂G]
-               A⌜R⋂G⌟ = overapproximate(A⌜R⋂G⌟, dirs)
-            end
+            A⌜R⋂G⌟ = apply_assignment(𝒫, assignment, R⋂G, oa=oa)
```

Comment:
same here: `A⌜R⋂G⌟ = apply(assignment, R⋂G)` doing the overapproximation below this line

Reply:
?

Reply:
Okay, I agree that the overapproximation should not have been outsourced.